src/spec.ts

```diff
@@ -355,3 +356,124 @@ export function isStacked(spec: TopLevel<FacetedCompositeUnitSpec>, config?: Con
   }
   return false;
 }
+
+export function selectedFields(spec: NormalizedSpec): string[] {
```

Comment:
Selected is a weird name. Should we better call this `usedFields`? Also, add a comment describing what this method does and when it is useful.